### PR TITLE
Mayank initial work

### PR DESCRIPTION
**Pull request by mayshri**

### CODE REVIEW

File: monitoring/online_evaluation.py [hunk 1]
```diff
@@ -0,0 +1,478 @@
...
+
+    return "Malformed", time_dt, user_id, None
+
+
+def get_consumer(no_mock_data=False, mock_fp="cache/mock_movielog.txt", seed=42, **kwargs):
+    """ Create (mock) Kafka stream consumer. """
+
+    if no_mock_data:
+
+            consumer = KafkaConsumer("movielog2",
```
Comment (gabrieltseng) [inline monitoring/online_evaluation.py:368]: I guess this is meant to be `movielog5`?

Comment (gabrieltseng) [reply monitoring/online_evaluation.py:368]: Also as we discussed all our evaluation metrics are stored in a `txt` file [here](https://github.com/COMP585Fall2022/Team-5/tree/main/metrics), so there shouldn't need to be a Kafka Consumer feeding anything into the influx database.

Comment (mayshri) [reply monitoring/online_evaluation.py:368]: Ok
Yes it will be log5 might be typo error, sorry

**Comment by gabrieltseng** [comment]
Thanks @mayshri ! A few general comments & thoughts:

1. As I mentioned in the comment, we only intended for the Grafana dash to read our online evaluation metrics, which are stored in `.txt` files in `metrics`. This should simplify this PR quite a bit
    - In general, we have worked out a pretty extensive [online evaluation](https://github.com/COMP585Fall2022/Team-5/blob/main/src/monitor/monitor.py#L19) system which it probably makes sense to build off
    
2. Is Grafana currently running on the server? There are a few unclear bits for deployment (e.g. where does the `apikey.txt` come from) - have you got it up and running? Could you share some screenshots?

3. There are a few big design decisions made here which we hadn't discussed (e.g. the introduction of new evaluation metrics, the storage of user information in addition to evaluation points). This is neat! but makes this PR huge and difficult to review - in the future it would be better to (a) discuss them in the group and (b) split this work into smaller PRs. What was the motivation for introducing this additional complexity to the PR?

**Pull request closed without merging**